src/highdicom/sr/templates/tid3700.py

```diff
@@ -0,0 +1,1213 @@
+            if not isinstance(ecg_findings, (list, tuple, set)):
+                raise TypeError(
+                    'Argument "ecg_findings" must be a sequence.'
+                )
+            for ecg_finding in ecg_findings:
+                if not isinstance(ecg_finding, ECGFinding):
+                    raise TypeError(
+                        'Items of argument "ecg_finding" must have ' +
+                        'type ECGFinding.'
+                    )
+                content.extend(ecg_finding)
+        if len(content) > 0:
+            item.ContentSequence = content
+        super().__init__([item])
+
+
+class ECGQualitativeAnalysis(Template):
+    """:dcm:`TID 3717 <part16/sect_TID_3717.html>`
+    ECG Qualitative Analysis
+    """
+
+    def __init__(
+        self,
+        ecg_finding_text: Optional[str] = None,
+        ecg_finding_codes: Optional[Sequence[ECGFinding]] = None
+    ) -> None:
+        item = ContainerContentItem(
+            name=codes.DCM.QualitativeAnalysis,
+            template_id='3717',
+            relationship_type=RelationshipTypeValues.CONTAINS
+        )
+        content = ContentSequence()
+        if ecg_finding_text is None and ecg_finding_codes is None:
+            raise ValueError(
+                'Either argument "ecg_finding_text" or "ECG_finding_code" ' +
+                'must be given.'
+            )
+        if ecg_finding_text is not None:
+            if not isinstance(ecg_finding_text, str):
+                raise TypeError(
+                    'Argument "ecg_finding_text" must have type str.'
+                )
+            ecg_finding_text_item = TextContentItem(
+                name=CodedConcept(
+                    value='271921002',
+                    meaning='ECG Finding',
+                    scheme_designator='SCT'
+                ),
+                value=ecg_finding_text,
+                relationship_type=RelationshipTypeValues.CONTAINS
+            )
+            content.append(ecg_finding_text_item)
+        if ecg_finding_codes is not None:
+            if not isinstance(ecg_finding_codes, (list, tuple, set)):
+                raise TypeError(
+                    'Argument "ecg_finding_code" must be a sequence.'
+                )
+            for ecg_finding_code in ecg_finding_codes:
+                if not isinstance(ecg_finding_code, ECGFinding):
+                    raise TypeError(
+                        'Items of argument "ecg_finding" must have ' +
+                        'type ECGFinding.'
+                    )
+                content.extend(ecg_finding_code)
+        if len(content) > 0:
+            item.ContentSequence = content
+        super().__init__([item])
+
+
+class SummaryECG(Template):
+    """:dcm:`TID 3719 <part16/sect_TID_3719.html>`
+    Summary, ECG
+    """
+
+    def __init__(
+        self,
+        summary: Optional[str] = None,
+        ecg_overall_finding: Optional[Union[Code, CodedConcept]] = None
```

Comment:
include link to [CID 3677](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_3677.html)